sites/gcweb-menu/inc-sitemenu.html

```diff
@@ -1,41 +1,39 @@
 <nav class="gcweb-menu" typeof="SiteNavigationElement">
```

Comment:
Do you mind to move the "inc-*" file into a new folder named "includes" and to remove the prefix? Thanks